tests/test_features/test_importances.py

```diff
@@ -248,6 +249,24 @@ def test_multi_coefs(self):
 
         npt.assert_equal(visualizer.feature_importances_.ndim, 1)
 
+    @pytest.mark.xfail(
+        sys.platform == 'win32', reason="images not close on windows"
+    )
+    def test_multi_coefs_stacked(self):
+        """
+        Test stack plot with multidimensional coefficients
+        """
+        X_iris, y_iris = load_iris(True)
```

Comment:
Very good test, thank you so much! Another thing we do in testing is use [`sklearn.datasets.make_classification`](http://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_classification.html) to specify exactly how the classification behaves with some fixed random state. The baseline image you created with Iris is a good test, though, so no need to jump through any other hoops!